browser/resources/settings/brave_leo_assistant_page/customization_subpage.html

```diff
@@ -0,0 +1,169 @@
+    icon="tune"
+    pref="{{prefs.brave.ai_chat.user_customization_enabled}}"
+    label="$i18n{braveLeoAssistantCustomizationEnabledLabel}">
+  </settings-toggle-button>
+</div>
+
+<div class="leo-customization-form leo-customization-padding">
+  <div class="input-container">
+    <div class="custom-label">
+      <div class="label-text">
+        <span>$i18n{braveLeoAssistantCustomizationNameLabel}</span>
+      </div>
+    </div>
+    <leo-input
+      placeholder="$i18n{braveLeoAssistantCustomizationNamePlaceholder}"
+      value="[[nameInput_]]"
+      on-change="onNameInputChanged_"
+      showErrors$="[[nameInputError_]]">
+      <div slot="errors" class="error-message" hidden$="[[!nameInputError_]]">
+        <leo-icon name="warning-triangle-filled"></leo-icon>
+        <span>$i18n{braveLeoAssistantCustomizationNameError}</span>
+      </div>
+    </leo-input>
+  </div>
+  <div class="input-container">
+    <div class="custom-label">
+      <div class="label-text">
+        <span>$i18n{braveLeoAssistantCustomizationJobLabel}</span>
+      </div>
+    </div>
+    <leo-input
+      placeholder="$i18n{braveLeoAssistantCustomizationJobPlaceholder}"
+      value="[[jobInput_]]"
+      on-change="onJobInputChanged_"
+      showErrors$="[[jobInputError_]]">
+      <div slot="errors" class="error-message" hidden$="[[!jobInputError_]]">
+        <leo-icon name="warning-triangle-filled"></leo-icon>
+        <span>$i18n{braveLeoAssistantCustomizationJobError}</span>
+      </div>
+    </leo-input>
+  </div>
+  <div class="input-container">
+    <div class="custom-label">
+      <div class="label-text">
+        <span>$i18n{braveLeoAssistantCustomizationToneLabel}</span>
+      </div>
+    </div>
+    <leo-input
+      placeholder="$i18n{braveLeoAssistantCustomizationTonePlaceholder}"
+      value="[[toneInput_]]"
+      on-change="onToneInputChanged_"
+      showErrors$="[[toneInputError_]]">
+      <div slot="errors" class="error-message" hidden$="[[!toneInputError_]]">
+        <leo-icon name="warning-triangle-filled"></leo-icon>
+        <span>$i18n{braveLeoAssistantCustomizationToneError}</span>
+      </div>
+    </leo-input>
+  </div>
+  <div class="input-container">
+    <div class="custom-label">
+      <div class="label-text">
+        <span>$i18n{braveLeoAssistantCustomizationOtherLabel}</span>
+      </div>
+    </div>
+    <leo-input
+      placeholder="$i18n{braveLeoAssistantCustomizationOtherPlaceholder}"
+      value="[[otherInput_]]"
+      on-change="onOtherInputChanged_"
+      showErrors$="[[otherInputError_]]">
+      <div slot="errors" class="error-message" hidden$="[[!otherInputError_]]">
+        <leo-icon name="warning-triangle-filled"></leo-icon>
+        <span>$i18n{braveLeoAssistantCustomizationOtherError}</span>
+      </div>
+    </leo-input>
+  </div>
+  <div class="actions">
+    <leo-button kind="outline" on-click="onSave" isDisabled$="[[isSaveDisabled_]]">$i18n{braveLeoAssistantCustomizationSaveButton}</leo-button>
```

Comment:
This is just an observation but in general changes to settings are instant and you don't have to press save to have them apply (unless they're in a dialog).

cc @aguscruiz for his thoughts on this